NitroxClient/GameLogic/Vehicles.cs

```diff
@@ -33,6 +34,7 @@ public Vehicles(IPacketSender packetSender, PlayerManager playerManager, IMultip
             this.packetSender = packetSender;
             this.playerManager = playerManager;
             this.multiplayerSession = multiplayerSession;
+            cyclops = null;
```

Comment:
Can we inject with the dependency framework?

Reply:
No we can´t it will error in cyclic dependency